tests/model_sdk_performance_test.go

```diff
@@ -28,6 +28,21 @@ type modelPerf struct {
 	*TestContext
 }
 
+func Test_SpecRamco(t *testing.T) {
```

Comment:
Probably shouldn't put this company name here.

Reply:
ah, you are correct, will fix